xDSCResourceDesigner.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.2.0.0'
+ModuleVersion = '1.2.1.0'
```

Comment:
Actually, per contribution guideline, I shouldn't be changing the module version
*DO NOT change the *.psd1 ModuleVersion (we will be updating this before releasing to the Gallery)*